### DISCUSSION TITLE
funny happened

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#404] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
someone sent a 4 petabyte zip bomb to my teacher and his computer fried lol

**Comment by Aka-but-nice** [comment]
Mine would too, a petabyte is 1,048,576 GB I only have 64 gigs

**Comment by rwg100** [comment]
im guessing it was 42.zip lol
if you extract the zip bomb and look at the contents you can tell its a zip bomb

**Reply by obkduishinijf** [reply]
he ain't smart enough to check it first, and yeah it was 42.zip. when he extracted all it literally killed the whole computer
